--- diffutils-2.7.2/Makefile.in.orig	Mon Sep 14 08:59:43 1998
+++ diffutils-2.7.2/Makefile.in	Tue Jul  3 19:52:17 2001
@@ -48,9 +48,9 @@
 exec_prefix = @exec_prefix@
 edit_program_name = sed '@program_transform_name@'
 
-bindir = $(exec_prefix)/bin
+bindir = @bindir@
 
-infodir = $(prefix)/info
+infodir = @infodir@
 
 DEFAULT_DIFF_PROGRAM = $(bindir)/`echo diff | $(edit_program_name)`
 DEFAULT_EDITOR_PROGRAM = ed
@@ -181,20 +181,16 @@
 
 install: all installdirs
 	for p in $(PROGRAMS); do \
-	  $(INSTALL_PROGRAM) $$p $(bindir)/`echo $$p | $(edit_program_name)`; \
+	  $(INSTALL_PROGRAM) $$p $(DESTDIR)$(bindir)/`echo $$p | $(edit_program_name)`; \
 	done
 	if test -f diff.info; then d=.; else :; d=$(srcdir); fi; \
-	$(INSTALL_DATA) $$d/diff.info $(infodir)/diff.info; \
-	if $(SHELL) -c 'install-info --version' >/dev/null 2>&1; \
-	then install-info --dir-file=$(infodir) $$d/diff.info; \
-	else :; \
-	fi
+	$(INSTALL_DATA) $$d/diff.info $(DESTDIR)$(infodir)/diff.info
 
 install-strip:
 	$(MAKE) INSTALL_PROGRAM='$(INSTALL_PROGRAM) -s' install
 
 installdirs:
-	$(SHELL) $(srcdir)/mkinstalldirs $(bindir) $(infodir)
+	$(SHELL) $(srcdir)/mkinstalldirs $(DESTDIR)$(bindir) $(DESTDIR)$(infodir)
 
 # We need more tests.
 check: $(PROGRAMS)
